pkg/client/userd/trafficmgr/traffic_manager.go

```diff
@@ -396,15 +406,14 @@ func connectMgr(c context.Context, cluster *k8s.Cluster, installID string, svc S
 		dlog.Errorf(c, "unable to get APIKey: %v", err)
 	}
 
-	ti, err := NewTrafficManagerInstaller(cluster)
-	if err != nil {
-		return nil, stacktrace.Wrap(err, "new installer")
+	dlog.Debug(c, "checking that traffic-manager exists")
+	existing, _, isManagerErr := helm.IsTrafficManager(c, cluster.ConfigFlags, cluster.GetManagerNamespace())
+	if isManagerErr == nil && existing == nil {
+		return nil, errcat.User.New("traffic manager not found, if it is not installed, please run 'telepresence helm install'")
 	}
```

Comment:
this was my first approach, but a test failed, i think it was because on one test we have some missing permissions, but we want the install to succeed, that's why I still attempted the install even if I got nil, nil. If I missed additional changes, pls disregard

Reply:
Yeah, I also changed the `IsTrafficManager` so that it actually returns an error if it runs into permission problems.